x-pack/plugins/secops/public/components/load_more_table/index.tsx

```diff
@@ -149,7 +164,21 @@ export class LoadMoreTable<T> extends React.PureComponent<BasicTableProps<T>, Ba
           </>
         )}
         <EuiTitle size="s">{title}</EuiTitle>
-        <EuiBasicTable items={pageOfItems} columns={columns} />
+        <EuiBasicTable
+          items={pageOfItems}
+          columns={columns}
+          onChange={onChange}
+          sorting={
+            sorting
+              ? {
+                  sort: {
+                    field: sorting!.field,
```

Comment:
You shouldn't need the `sorting!.field` here since you're checking for `sorting` existing and not being null above. TypeScript is smart enough to do that for us and reduces our bangs.

Otherwise if it still complains you just have to change it out for `sorting != null ?` and TypeScript is smart enough.